packages/arb-bridge-eth/contracts/inbox/GlobalFTWallet.sol

```diff
@@ -69,13 +86,74 @@ contract GlobalFTWallet {
         return wallet.ftList[index - 1].balance;
     }
 
+    function isPairedContract(address _tokenContract, address _chain)
+        public
+        view
+        returns (bool)
+    {
+        return
+            pairedContracts[_tokenContract].connectedRollups[_chain] ==
+            PairingStatus.Paired;
+    }
+
+    function requestPairing(address _tokenContract, address _chain) internal {
```

Comment:
It looks like this only lets you try to pair once, and if that fails, you'll never be able to re-try the pairing.  Is there a way to deal with failed attempts that will be more robust?